### DISCUSSION TITLE
My Achievements is not show

### DISCUSSION THREAD

**Discussion by felipsbreno** [community/community#37577] [Other Feature Feedback, Questions, & Ideas] [answered]
My Achievements is not show, i was in my profile and enabled to show, but not show. More some people with this problem ?? 

![image](https://user-images.githubusercontent.com/50807243/198417152-f683f705-eb6b-48c9-90b7-c9f570647ebd.png)

**Comment by radar-blip** [comment, marked as answer]
Did you remember to click on `Update preferences` ?

Also, try this link to unhide whichever achievements you want showing: https://github.com/felipsbreno?tab=achievements

**Reply by felipsbreno** [reply]
Thank you @linuxlawson , it returned the showing my achievements in my profile

**Reply by radar-blip** [reply]
You're welcome! Glad I could help!!

**Reply by nikolast178** [reply]
I have the same problem. I try to use "/nikolast178?tab=achievements" but I get **404**.

**Reply by torikallcode** [reply]
Same, mine is like that too

**Comment by tinaponting** [comment]
Nice:)

**Comment by haiderali27** [comment]
I tried it but I still can't see my achievements. They used to be there. They are gone now.

**Reply by chouchouji** [reply]
Me too! I try to click `update preferences`、 relogin and clear cache, but all don't work.

**Reply by haiderali27** [reply]
I have found a solution,  from here
https://github.com/orgs/community/discussions/37577#discussioncomment-3987277
Go to your url like this, there you can get it back
https://github.com/{username}?tab=achievements 
For E.g this will work for you. 

https://github.com/chouchouji?tab=achievements

**Reply by chouchouji** [reply]
It works, wow, thank you very much!!!

**Reply by hurrainjhl** [reply]
the link is show error 404 foe me.
